src/App.tsx

```diff
@@ -19,10 +39,25 @@ class App extends React.Component {
           >
             Learn React
           </a>
+          <p>Toggle this on and off to test out Redux:</p>
+          <p>
+            <button
+              onClick={this.handleToggleClick}
+              style={{ padding: '32px' }}
+            >
+              {toggledOn ? 'OFF' : 'ON'}
+            </button>
+          </p>
         </header>
       </div>
     );
   }
 }
 
-export default App;
+const mapStateToProps = (state: ApplicationState) => {
+  return {
+    toggledOn: state.example.toggledOn,
```

Comment:
I did a test here where I added an additional property to this object, and the TS compiler didn't complain. Shouldn't we be trying to enforce type safety in this case as well, or is that just something we choose not to worry about?

Reply:
Good catch. I guess when I tried it, I only changed the value of `toggledOn` (which produced an error). I added a type hint for the return type of `PropsFromState` which addresses it.